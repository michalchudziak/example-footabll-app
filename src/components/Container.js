/* @flow */
import styled from 'styled-components/native';

const Container = styled.View`
  flex: 1;
  align-items: center;
`;

export default Container;
