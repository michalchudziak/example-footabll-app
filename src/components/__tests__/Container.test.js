/* @flow */
import React from 'react';
import renderer from 'react-test-renderer';
import Container from '../Container';

describe('<Container />', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Container />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
