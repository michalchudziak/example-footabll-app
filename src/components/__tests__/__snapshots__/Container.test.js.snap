// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
      undefined,
    ]
  }
/>
`;
